### ISSUE TITLE
gtfs to geojson

### ISSUE THREAD

**Issue by Alirezat0** [BlinkTagInc/gtfs-to-geojson#16]
Hi Mr.Nee,
I have some issues with importing gtfs data to Mapbox
firstly i should convert a zip gtfs data that I have to geojson file then I should add it to a map 
i created a config.json file then when I want to run (gtfs-to-geojson --configPath /path/to/your/config.json) i will get this error :
in the following, I will send a pic of my error and my gtfs data for SanFransico
![Screenshot (287)](https://user-images.githubusercontent.com/99495220/188850221-d4a10034-7481-46ba-a06c-fa3a2e277775.png)
[all-transit-master.zip](https://github.com/BlinkTagInc/gtfs-to-geojson/files/9505128/all-transit-master.zip)

**Comment by brendannee** [comment]
It looks like there is an unknown ? character in your configPath command, right before the `C:\`. Try running just `gtfs-to-geojson` without any other parameters from the directory that has your config.json in it, this should automatically use the config.json file in that folder without you needing to specify the path.

**Comment by Alirezat0** [comment]
Hi Mr.Nee,
yes, thanks that were an error I deleted the (?) now I got some other error
and I have another question what exactly does this code(gtfs-to-geojson --configPath /path/to/your/config.json) do, should I first convert gtfs to json file then by this library convert it to geojson?
or it converts my zipped gtfs to geojson directly?
thanks for your time.
![Screenshot (288)](https://user-images.githubusercontent.com/99495220/189080726-35910a5e-a882-4e45-a876-348ef195e10e.png)
